bindings/wasm/examples-account/README.md

```diff
@@ -0,0 +1,37 @@
+![banner](./../../../.meta/identity_banner.png)
+
+
+## IOTA Identity Account Examples
+
+This folder provides code examples for you to learn how the IOTA Identity WASM bindings for the Account can be used in JavaScript/Typescript.
+
+The examples are written in Typescript and can be run independently with Node.js. 
+
+### Node.js
+In order to run the examples in Node.js make sure to install the dependencies: 
+```bash
+npm install
+```
+
+Then run the example using:
+```bash
+npm run example:account -- <example-name>
+```
+
+For instance, to run the `create_did` example use:
+```bash
+npm run example:node -- create_did
+```
+
+
+| # | Name | Details |
+| -------- | -------- | -------- |
+| 1 |[create_did](src/create_did.ts)|Generates and publishes a DID Document, the fundamental building block for decentralized identity.|
+|2| [manipulate_did](src/manipulate_did.ts)|  Adds verification methods and service endpoints to a DID Document and updates an already existing DID Document.|           
+|3| [lazy](src/lazy.ts)| Manipulates a DID Document and publishes multiple changes to the tangle at once.|
+|4| [signing](src/signing.ts) | Creates a DID with a verification method, signs a verifiable credential, verifies the signature.|
+|5| [config](src/config.ts) | Applies different account configurations|
+|6| [unchecked](src/unchecked.ts) | Updates an Identity without validation and publishes it to the tangle.|
+## Browser
+Although the examples should work in browser environment, we don't provide a browser project as for now.
```

Comment:
It's not done, maybe in a future issue depending on if we decide to move them with the low level examples. The code should also be valid JavaScript code. (it also can be transpiled to JavaScript if we decide to add TypeScript code, this shouldn't be complicated). In my opinion, using TypeScript by default is really important because we rely heavily on types. This also could encourage developers to support it if they're new to web development.

Reply:
Except we can't test the Account Typescript examples in the browser now, leaving us open to them simply not working in that environment. We have had numerous instances of the Wasm bindings working in Node.js just fine and breaking in the browser or vice-versa.